contracts/libs/0.8.x/BytecodeStorage.sol

```diff
@@ -31,9 +32,11 @@ library BytecodeStorage {
     // 53             | 20   | 73           | the 20 bytes used for storing the deploying contract's address         //
     //---------------------------------------------------------------------------------------------------------------//
     // Define the offset for where the "logic bytes" end, and the "data bytes" begin. Note that this is a manually
-    // calculated value, and must be updated if the above table is changed. It is expected that tests will fail if
-    // this value is not updated.
+    // calculated value, and must be updated if the above table is changed. It is expected that tests will fail
+    // loudly if these values are not updated in-step with eachother.
     uint256 internal constant DATA_OFFSET = 73;
+    uint256 internal constant ADDRESS_LENGTH = 20;
+    uint256 internal constant ADDRESS_OFFSET = 53; // DATA_OFFSET - ADDRESS_LENGTH
```

Comment:
nit: we could calculate DATA_OFFSET in a constructor now, and declare it immutable.

Reply:
oops - forgot this is a library and no constructors allowed. ignore this comment!